editoast/src/modelsv2/infra.rs

```diff
@@ -243,6 +245,22 @@ impl Infra {
         let voltages = sql_query(query).load::<Voltage>(conn).await?;
         Ok(voltages)
     }
+
+    pub async fn get_speed_limit_tags(
+        conn: &mut DbConnection,
+        infra_id: i64,
+    ) -> Result<Vec<SpeedLimitTags>> {
+        let speed_limits_tags = sql_query(
+            "SELECT DISTINCT jsonb_object_keys(data->'speed_limit_by_tag') AS tag
+            FROM infra_object_speed_section
+            WHERE infra_id = $1
+            ORDER BY tag",
```

Comment:
Do we want to homogeneize the way we encode SQL requests. In the previous PR #7431, the SQL requests were in external files. It has interesting properties for developers like syntax highlighting which might help catch syntax errors early. But maybe it's more important for big SQL requests, and not so much for small one?

cc @leovalais

Reply:
I'm ok with having it like this *for now* in order to move diesel out of the views asap. 

My take on this is that we shouldn't have raw SQL at all anymore. We have the ModelV2 API and diesel's DSL that ensure are requests are type-checked and we should use that as much as possible.

Reply:
For example, this request can be rewritten with `List::list` + 'map'. (Admittedly less efficient because we deserialize the whole model.)

Reply:
I created an external file as discussed @woshilapin.